automation/run-local-utests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+TOXWORKDIR="/tmp/.tox-nmstate"
+
+mkdir -p "$TOXWORKDIR"
+
+docker run \
+       --rm \
+       -ti \
+       -v $PWD:/workspace/nmstate:Z \
+       -v $TOXWORKDIR:$TOXWORKDIR:Z \
+       nmstate/centos7-utest \
```

Comment:
IMHO it would be better to not abbreviate unit test as utest.

Reply:
Well, it is not only unit tests, it also needs to cover lints (static analysis).

Reply:
This does not make utests a better name :-P

Reply:
The maker has usually the privilege to name it if it is not completely wrong.

Reply:
You said that unit test is wrong, so abbreviating it as utest is wrong as well....